crates/interpreter/src/instructions/bitwise.rs

```diff
@@ -257,19 +257,18 @@ mod tests {
         }
 
         for test in test_cases {
-            host.clear();
             push!(interpreter, test.value);
             push!(interpreter, test.shift);
-            shl::<Interpreter, DummyHost<DefaultEthereumWiring>>(&mut interpreter, &mut host);
+            shl::<EthInterpreter, DummyHost>(&mut interpreter, &mut host);
```

Comment:
```suggestion
            shl(&mut interpreter, &mut host);
```

Check if we can go without it

Reply:
There are few more examples of this in PR

Reply:
It works. Thanks for the suggestion.